shared_model/backend/protobuf/queries/proto_account_asset_pagination_meta.hpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_SHARED_PROTO_MODEL_QUERY_ACCOUNT_ASSET_PAGINATION_META_HPP
+#define IROHA_SHARED_PROTO_MODEL_QUERY_ACCOUNT_ASSET_PAGINATION_META_HPP
+
+#include "backend/protobuf/common_objects/trivial_proto.hpp"
+#include "interfaces/common_objects/types.hpp"
+#include "interfaces/queries/account_asset_pagination_meta.hpp"
```

Comment:
Maybe `#include "interfaces/queries/account_asset_pagination_meta.hpp"` should be the first, and separated with a newline?